plugins/modules/system/vdo.py

```diff
@@ -437,7 +437,7 @@ def add_vdooptions(params):
     if params.get('physicalthreads') is not None:
         options.append("--vdoPhysicalThreads=" + params['physicalthreads'])
 
-    return vdocmdoptions
+    return options
```

Comment:
Maybe also remove the `vdocmdoptions` variable at the beginning of this function.